packages/material-ui-benchmark/package.json

```diff
@@ -28,7 +28,6 @@
     "@emotion/core": "^10.0.0",
     "@emotion/styled": "^10.0.0",
     "benchmark": "^2.1.4",
-    "emotion-server": "^10.0.6",
```

Comment:
This is a migration/compat package only: https://emotion.sh/docs/ssr Not sure if a benchmark for those kind of packages is usefull? @oliviertassinari

Reply:
Wait, what it's legacy? How are we supposed to handle interoperability specificity then?
Let's say I have a core component X that uses emotion but I also have a component Y that extends the style of X using styled-components. If I understand it correctly, styled-components with injecting the CSS in the head when emotion will inject the CSS in the body. The SSR output specificity will be wrong. cc @tkh44.

Reply:
Not legacy. Just "compatibility and migration purposes". I don't think that applies to the benchmark in question. It would however apply to the use case you mentioned I think.

Reply:
Without to mention this concerns: https://github.com/whatwg/html/issues/1605#issuecomment-235956455

Reply:
We need to benchmark the solution has a whole, with a fully working solution.

Reply:
I don't understand why you derail this to emotion concerns? The package specifically states that this is a package for compatibility concerns. If you think that we need to benchmark this solution then I have no objections. Let's fix the warning and keep the benchmark then.

Criticizing emotion architectural choices here is misplaced. If you have problems with emotions architectural choices you can open issues in their repo.